js_modules/dagit/packages/core/src/asset-graph/AssetNode.tsx

```diff
@@ -135,21 +135,31 @@ export const AssetNode: React.FC<{
 
 export const AssetNodeMinimal: React.FC<{
   selected: boolean;
-  style?: CSSProperties;
-}> = ({selected, style, children}) => {
+  definition: AssetNodeFragment;
+}> = ({selected, definition}) => {
   return (
-    <AssetNodeContainer $selected={selected} style={{position: 'absolute', borderRadius: 12}}>
+    <AssetNodeContainer
+      $selected={selected}
+      style={{
+        position: 'absolute',
+        borderRadius: 12,
+        outlineOffset: 2,
+        outlineWidth: 4,
+      }}
```

Comment:
Worth pushing this down into the `AssetNodeContainer` styled-component rule?

Reply:
Ahh good call yeah let me refactor this a bit!